lib/validation/validators.js

```diff
@@ -85,21 +85,27 @@ module.exports = Validators = {
   },
 
   // eslint-disable-next-line no-inline-comments, spaced-comment
-  before(value/*, [diff, unit][, diff, unit]*/) {
-    const args = [].slice.call(arguments, 1);
-    let date = moment(value, dateFormat);
-    let diff;
-    let unit;
-    while (args.length) {
-      diff = args.shift();
-      unit = args.shift() || 'years';
-      date = date.add(diff, unit);
+  before(value, date/*, [diff, unit][, diff, unit]*/) {
+    let valueDate = moment(value, dateFormat);
+    let comparator;
+    if (arguments.length === 2) {
+      comparator = date;
+    } else {
+      comparator = moment();
+      const args = [].slice.call(arguments, 1);
+      let diff;
+      let unit;
+      while (args.length) {
+        diff = args.shift();
+        unit = args.shift() || 'years';
+        valueDate = valueDate.add(diff, unit);
+      }
     }
-    return value === '' || Validators.date(value) && date.isBefore(moment());
+    return value === '' || Validators.date(value) && valueDate.isBefore(comparator);
   },
 
   after(value, date) {
-    let test = moment(value, dateFormat);
```

Comment:
changed the name of test is a variable, because it name confused me.  I couldn't think of a better variable name than valueDate, happy to take suggestions